### ISSUE TITLE
AST for P

### ISSUE THREAD

**Issue by pinzaghi** [p-org/P#359]
I'm doing static analysis over P programs, so I started generating a parse tree using the ANTLR tool with the [grammar files](https://github.com/p-org/P/tree/master/Src/PCompiler/CompilerCore/Parser). This tool builds a visitor for the parser tree, which is very fine-grained for my requirements. 

Is there any AST already done for P in python or other language? I could do it by myself, but if there is already something to start with, it would save me some time.

**Comment by ankushdesai** [comment]
Hi @pinzaghi, What do you mean by there being an AST in any other language? 
We only have one parser for P language right now in C# using ANTLR.

**Comment by pinzaghi** [comment]
I want to parse P code and obtain an AST, ideally in python. But from what you say, there is only work made in C#. Just wanted to know before starting developing something.
